src/osyris/__init__.py

```diff
@@ -7,4 +7,4 @@
 from .units import units
 from .core import Array, Datagroup, Dataset, Plot, Vector
 from .plot import histogram1d, histogram2d, scatter, map, plot
-from .spatial import extract_box, extract_sphere
+from .spatial import extract_box, extract_sphere, change_origin, change_basis
```

Comment:
I don't know if all these should be imported at the top level. Maybe we could import the `spatial` module instead?
Once we make something available at the very top level, it's much more difficult to remove it later.

Reply:
> Maybe we could import the spatial module instead?

What do you mean by that? A simple `import .spatial` ?